test/shlibloadtest.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
That copyright line from Oracle is confusing, and I think we should try to avoid it. It claims _something_ in that file is copyright by Oracle and that we don't have a license for it.

In any case, you should not update copyright years that don't belong to you or the OpenSSL project.